modules/social_features/social_user/social_user.oauth2_scopes.yml

```diff
@@ -0,0 +1,145 @@
+"viewer":
```

Comment:
Hi @Kingdutch,

Could you maybe elaborate why this was added? 🤔

Reply:
@ronaldtebrake Looks like I accidentally did `git add modules` while this was an untracked file in my workdir and it got added to this PR 😅 

The file should be removed from the PR (I've made a local copy of it)